src/providers/export/tensorFlowRecords/tensorFlowHelpers.test.ts

```diff
@@ -37,4 +37,10 @@ describe("TFRecords Helper Functions", () => {
             expect(textEncode("ABC123")).toEqual(new Uint8Array([65, 66, 67, 49, 50, 51]));
```

Comment:
make `"ABC123"` and the array contants

Reply:
I don't see the point here.  This string/array is only used once in this call.  Specially in test I see this more synthetic.